tests/lora/utils.py

```diff
@@ -85,8 +85,13 @@ class PeftLoraLoaderMixinTests:
     unet_kwargs = None
     transformer_cls = None
     transformer_kwargs = None
+    vae_cls = AutoencoderKL
```

Comment:
Need different VAE for CogVideoX but set it to this as default for all other pipelines currently